core/lib/presentation/views/html_viewer/html_content_viewer_widget.dart

```diff
@@ -55,7 +55,7 @@ class _HtmlContentViewState extends State<HtmlContentViewer> {
   }
 
   String _generateHtmlDocument(String content) {
-    final htmlTemplate = generateHtml(content, hideScrollBar: true);
+    final htmlTemplate = generateHtml(content, hideScrollBar: false);
```

Comment:
1. Could you please explain more about this change?
2. Like the demo video, I feel that when scrolling horizontally it is not very smooth. Is there any way to improve it?

Reply:
1. Yeah! 
hidden - The overflow is clipped, and the rest of the content will be invisible (it mean cant scroll to visible all content)
auto - The overflow is clipped, and a scrollbar is added to see the rest of the content, but it adds scrollbars only when necessary
You can check link in here: 
 https://www.w3schools.com/css/css_overflow.asp
 2. Yeah I add some properties for smooth scroll animation and re-upload video. Please check this